src/twisted/newsfragments/9442.bugfix

```diff
@@ -0,0 +1 @@
+twisted.python.htmlizer.TokenPrinter now explicitly works on bytestrings.
```

Comment:
This is definitely true and the underlying cause, but it'd be good to mention `lastColorized`, because it is the issue that led to the report.

Reply:
That's in another topfile :)

Reply:
Whoops, sorry!  So it is!

Nothing like a fix of several fixes - yay!